src/main/webapp/css/srp_style.css

```diff
@@ -43,3 +56,22 @@
   margin-top: 0;
```

Comment:
I'd add left and right margins to .card code so the code text doesn't stick to the side of the grey area